site/api/prisma/migrations/20220313101843_group_person_datamodel_updates/migration.sql

```diff
@@ -0,0 +1,87 @@
+-- CreateEnum
+CREATE TYPE "GroupGender" AS ENUM ('MALE', 'FEMALE', 'COED');
+
+-- AlterTable
+ALTER TABLE "groups" ADD COLUMN     "added_by_id" INTEGER,
+ADD COLUMN     "company_id" INTEGER,
+ADD COLUMN     "creation_date" TIMESTAMP(3),
+ADD COLUMN     "discord" TEXT,
+ADD COLUMN     "facebook" TEXT,
+ADD COLUMN     "fan_cafe" TEXT,
+ADD COLUMN     "fan_name" TEXT,
+ADD COLUMN     "gender" "GroupGender",
+ADD COLUMN     "instagram" TEXT,
+ADD COLUMN     "melon" TEXT,
+ADD COLUMN     "spotify" TEXT,
+ADD COLUMN     "tiktok" TEXT,
+ADD COLUMN     "twitter" TEXT,
+ADD COLUMN     "vlive" TEXT,
+ADD COLUMN     "website" TEXT,
+ADD COLUMN     "youtube" TEXT;
+
+-- AlterTable
+ALTER TABLE "image_reports" RENAME COLUMN "actionedAt" to "actioned_at";
+
+-- AlterTable
+ALTER TABLE "persons" ADD COLUMN     "added_by_id" INTEGER,
+ADD COLUMN     "discord" TEXT,
+ADD COLUMN     "eye_color" TEXT,
+ADD COLUMN     "facebook" TEXT,
+ADD COLUMN     "fan_cafe" TEXT,
+ADD COLUMN     "height" INTEGER,
+ADD COLUMN     "instagram" TEXT,
+ADD COLUMN     "melon" TEXT,
+ADD COLUMN     "native_name" TEXT,
+ADD COLUMN     "spotify" TEXT,
+ADD COLUMN     "tiktok" TEXT,
+ADD COLUMN     "twitter" TEXT,
+ADD COLUMN     "vlive" TEXT,
+ADD COLUMN     "website" TEXT,
+ADD COLUMN     "youtube" TEXT;
+
+-- AlterTable
+ALTER TABLE "user_restrictions" RENAME COLUMN "associatedEntityId" TO "associated_entity_id";
+
+-- CreateTable
+CREATE TABLE "person_nationalities" (
+    "id" SERIAL NOT NULL,
+    "country" TEXT NOT NULL,
+    "person_id" INTEGER NOT NULL,
+    "added_by_id" INTEGER,
+    "created_at" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    CONSTRAINT "person_nationalities_pkey" PRIMARY KEY ("id")
+);
```

Comment:
tbh, i would make this just on the Idol table.. we dont care who added it, we dont care about it alone.. When we want to query for idols from a specific country, we could do it on the idol.

Reply:
well some idols have more than 1 nationality so it needs to be a 1:M relationship. Might as well add some more metadata while we're at it

Reply:
This is definitely something we want to query by at some point so I don't think it makes sense to denormalize it into an array field either

Reply:
Im just not sure this is interesting enough to our users :D 

If we go for the more agnostic approach, this is fine

Reply:
It's not for the users tho, I would much rather save too much info than too little. It's good to be able to know who made what change. That can also just be tracked in audit logs I suppose but we don't really have that setup yet